ScalaApp/scala_settings.py

```diff
@@ -23,6 +23,9 @@
 MINIMUM_BALANCE_FOR_DINING_SLOT_CLAIM = Decimal('-2.00') - KITCHEN_COST
 MINIMUM_BALANCE_FOR_USER_TRANSACTION = Decimal('0.00')
 
+# Except associations from bottom limit by association id
+MINIMUM_ASSOCIATION_EXCEPTIONS = []
```

Comment:
For optimal use this should be softcoded not hardcoded. I.e. This is a setting that should be in the Association model stored on the database not somewhere in the code.

Reply:
Do you want it that way? It's probably not a change that will be made lightly and it can be compared with e.g. the kitchen cost setting which is also hardcoded. It's a setting which means that it's meant to be changeable for each deployment.

Reply:
Changing a limit is indeed not something that should be taken lightly, however, it should also not be something that should potentially need to go through the ICT commission in my opinion. If Q wants an unlimited balance, I'm gonna assume other associations also want some custom arrangement (possibly with higher, but not unlimited depths, though I still need to inquire that). Thus any adjustment should be visible in the system itself instead of in the code and should be able to be adjusted by the Scala board directly if neccessary.
Also, I personally think it is weird to hardcorde an exception for a specific instance in the database table.

Reply:
Now added as a tickbox in the admin panel